packages/web-twig/src/Resources/components/Modal/Modal.stories.twig

```diff
@@ -17,9 +17,22 @@
             footerElement.classList.add(`ModalFooter--${event.target.value}`);
         };
 
+        const toggleDockOnMobile = (selector, dependingInputSelector) => {
+            const dependingInputElement = document.querySelector(dependingInputSelector);
+            const dependingLabelElement = dependingInputElement.closest('label');
+
+            document.querySelector(selector).classList.toggle('ModalDialog--dockOnMobile');
```

Comment:
this could be also in constant as `dependingInputElement` and `dependingLabelElement`